### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/layout_catalog/
https://pavel-gutsal.github.io/layout_catalog/report/html_report/

data-qa="card-hover" - responsible for testing hover on link feature ;
it is present on each link elemet 
yet test sometimes passes successfully or fails without any changes to the code 
on hover effect on site itself is operational

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,377 @@
...
+        </div>
+        <div 
+          class="main__card main__card--margin"
+        >
+          <div class="main__div main__div--center">
+            <img 
+            src="./images/imac.jpeg" 
+            alt="picture" 
+            class="main__img"
+            >
```
Comment (ArturErmolenko) [inline src/index.html:159]:
```suggestion
            <img 
              src="./images/imac.jpeg" 
              alt="picture" 
              class="main__img"
            >
```

File: src/styles/blocks/stars.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    background-image: url("../images/star-active.svg");
+  }
+  
+  &--star-4 &__star:nth-child(-n+4) {
+    background-image: url("../images/star-active.svg");
+  }
+  
+  &--star-5 &__star:nth-child(-n+5) {
+    background-image: url("../images/star-active.svg");
+  }
```
Comment (ArturErmolenko) [inline src/styles/blocks/stars.scss:33]: селекторы с одинаковыми стилями лучше записать через запятую
```
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
   background-image: url("/images/star-active.svg");
 }
```

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Good job 👍 
just small fix

**Comment by pavel-gutsal** [comment]
all issues are fixed

File: src/index.html [hunk 3]
```diff
@@ -7,6 +7,377 @@
...
+        <div 
+          class="main__card main__card--margin"
+          data-qa="card"
+        >
+          <div class="main__div main__div--center">
+            <img 
+            src="./images/imac.jpeg" 
+            alt="picture" 
+            class="main__img"
+            >
```
Comment (vkryvytskyy) [inline src/index.html:83]:
```suggestion
            <img 
              src="./images/imac.jpeg" 
              alt="picture" 
              class="main__img"
            >
```

File: src/index.html [hunk 4]
```diff
@@ -7,6 +7,377 @@
...
+          </div>
+          <h1 class="main__header main__header--margin_bottom">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <h2 class="main__common-text main__common-text--margin_bottom">
+            Код товара: 195434
+          </h2>
+          <div class="main__reviewes main__reviewes--margin_bottom">
```
Comment (vkryvytskyy) [inline src/index.html:91]:
```suggestion
          <div class="main__reviews reviews">
```
why do you need separate class to margin-bottom? If you need set margins, set them in main__reviews and set own styles for reviews block in reviews class.
`main__reviews` describes positioning of element in the `main` block
`review` describes own styles for element

File: src/styles/blocks/mainBlock.scss [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $lightGray;
+  }
+
+  &__price ,
+  &__reviewes {
+    display: flex;
+    flex-direction: row;
```
Comment (vkryvytskyy) [inline src/styles/blocks/mainBlock.scss:67]: this is default value for flex-direction, should be ommited

**Review by vkryvytskyy — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
https://pavel-gutsal.github.io/layout_catalog/
https://pavel-gutsal.github.io/layout_catalog/report/html_report/
all issues were fixed

File: src/index.html [hunk 6]
```diff
@@ -7,6 +7,373 @@
...
+        </div>
+        <div 
+          class="main__card main__card--margin"
+        >
+          <div class="main__div main__div--center">
+            <img 
+            src="./images/imac.jpeg" 
+            alt="picture" 
+            class="main__img"
+            >
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:270]:
```suggestion
            <img 
              src="./images/imac.jpeg" 
              alt="picture" 
              class="main__img"
            >
```
fix everywhere

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Screenshot 2022-06-13 at 14 33 14

check your demos

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**